src/parser/jobs/sch/modules/DoTs.js

```diff
@@ -5,19 +5,25 @@ import {ActionLink, StatusLink} from 'components/ui/DbLink'
 import ACTIONS from 'data/ACTIONS'
 import STATUSES from 'data/STATUSES'
 import CoreDoTs from 'parser/core/modules/DoTs'
-import {Rule, Requirement} from 'parser/core/modules/Checklist'
+import {TieredRule, TARGET, Requirement} from 'parser/core/modules/Checklist'
 import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
 import DISPLAY_ORDER from './DISPLAY_ORDER'
 
-// In ms
-const CLIPPING_SEVERITY = {
-	1000: SEVERITY.MINOR,
-	10000: SEVERITY.MEDIUM,
-	30000: SEVERITY.MAJOR,
+// In seconds
+const SEVERITIES = {
+	CLIPPING: {
+		6: SEVERITY.MINOR,
+		9: SEVERITY.MEDIUM,
+		12: SEVERITY.MAJOR,
+	},
+	UPTIME: {
+		84: TARGET.WARN,
+		91: TARGET.SUCCESS,
+	},
 }
 
 export default class DoTs extends CoreDoTs {
-	static handle = 'dots'
+	static handle = 'Biolysis'
```

Comment:
Very nitpicky, but our handles are usually `lowerCamelCase` by convention, so this should be changed to `biolysis`.

Reply:
Not a nitpick if it's a convention – fix'd :)